.github/workflows/rust.yml

```diff
@@ -61,6 +57,7 @@ jobs:
       - name: Checkout branch
         uses: actions/checkout@v2
 
+      # This can be removed once https://github.com/actions-rs/toolchain/pull/209 is merged and released
```

Comment:
Once this is merged we can remove the `toolchain` parameter from `actions-rs/toolchain@v1` as it will read from the project `rust-toolchain.toml` file

Reply:
Should we open an issue instead to have better visibility of when we should do this?

Reply:
I'm a bit wary of creating issues to track on-going tasks of other repos but no big opinion on this one.